tests/election.py

```diff
@@ -70,6 +70,7 @@ def run(args):
                         "id": current_term,
                         "msg": "This log is committed in term {}".format(current_term),
                     },
+                    None,
                     True,
                 )
```

Comment:
Rather than adding a `None` parameter and relying on order, we should use the arg's name (`expected_result = True`)